docker-compose.yml

```diff
@@ -21,20 +21,12 @@ services:
       context: .
       dockerfile: ./docker/dev/Dockerfile
     command: ["gunicorn", "busy_beaver:app", "-b", "0.0.0.0:5100", "--reload", "--timeout",  "100000"]
+    env_file: .env
     environment:
```

Comment:
I will be using YAML anchoring for the task queue containers (which require the same vars) so we don't have to keep repeating this